modules/pom.xml

```diff
@@ -0,0 +1,395 @@
+                                            <property name="jboss.folder" value="${JBOSS_FOLDER}" />
+                                            <target name="copy-server-dir"/>
+                                        </ant>
+                                    </target>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+
+            <modules>
+                <module>testcases/jdkAll/Wildfly</module>
+            </modules>
+
+        </profile>
+
+        <profile>
+            <id>wildfly.jdk8.testsuite</id>
+            <activation>
+                <property>
+                    <name>wildfly-jdk8</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>prepare</id>
+                                <inherited>false</inherited>
+                                <phase>generate-resources</phase>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <configuration>
+                                    <target>
+                                        <ant antfile="${basedir}/src/test/scripts/copy-server-dir.xml">
+                                            <property name="todir" value="${basedir}/wildfly/build/target/jbossas" />
+                                            <property name="jboss.folder" value="${JBOSS_FOLDER}" />
+                                            <target name="copy-server-dir"/>
+                                        </ant>
+                                    </target>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+
+            <modules>
+                <module>testcases/jdk8/Wildfly</module>
+            </modules>
+
+        </profile>
+
+        <!-- Replicating the profiles in order to be activated via file existence. This section should be removed when multiple profile activation is supported by maven.-->
+        
+        <profile>
+            <id>eap.jdk8.testsuite.onserver</id>
+            <activation>
+                <file>
+                    <exists>../../eap-jdk8-testsuite</exists>
+                </file>
+            </activation>
+
+            <properties>
+                <JBOSS_FOLDER>../../../build/target/jboss-as-7.5.6.Final-redhat-SNAPSHOT/</JBOSS_FOLDER>
```

Comment:
The other solution is to configure it as an env variable it teamcity project agent. But since the folder name depends on the version we would have to do it for every version. (Or we could change the name of the produced folder to something independent of the version and configure the env variable in teamcity just once.)